pkg/kubelet/logs/container_log_manager.go

```diff
@@ -415,13 +415,17 @@ func (c *containerLogManager) removeExcessLogs(logs []string) ([]string, error)
 func (c *containerLogManager) compressLog(log string) error {
 	r, err := c.osInterface.Open(log)
 	if err != nil {
-		return fmt.Errorf("failed to open log %q: %v", log, err)
+		return fmt.Errorf("failed to open log %q: %w", log, err)
 	}
 	defer r.Close()
+	logInfo, err := os.Stat(log)
+	if err != nil {
+		return fmt.Errorf("failed to get log info %q: %w", log, err)
```

Comment:
nit: Stat error message already contains file name, I believe, you don't need to repeat it.
```suggestion
		return fmt.Errorf("failed to stat log file: %w", err)
```